modules/machineset-yaml-rhv.adoc

```diff
@@ -0,0 +1,110 @@
+<16>// Module included in the following assemblies:
+//
+// * machine_management/creating-infrastructure-machinesets.adoc
+// * machine_management/creating_machinesets/creating-machineset-rhv.adoc
+
+[id="machineset-yaml-rhv_{context}"]
+=  Sample YAML for a MachineSet Custom Resource on {rh-virtualization}
+
+This sample YAML defines a machine set that runs on
+{rh-virtualization} and creates nodes that are labeled with
+`node-role.kubernetes.io/<node_role>: ""`
+
+In this sample, `<infrastructure_id>` is the infrastructure ID label that is based on the cluster ID that you set when you provisioned the cluster, and `<role>` is the node label to add.
+
+[source,yaml,subs="+quotes"]
+----
+    apiVersion: machine.openshift.io/v1beta1
```

Comment:
This YAML example is starting indented by 4 spaces already, can you shift it left to start at the first column?

Reply:
fixed.